### PR TITLE
Update Security policy to be more precise about vulns

### PR DESCRIPTION
**Pull request by ret2libc**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -19,3 +19,5 @@ If you have not received a reply to your email within 48 hours, or have not hear
 - Inform the team over the [public chats](https://rizin.re/#community) that you sent a message regarding a security issue.
 
 **Important:** Don't disclose any information regarding the issue itself in the public chats.
+
+Given the current usage of Rizin and its libraries as a standalone Command Line Interface (CLI) tool, the security team does not consider as vulnerabilities those issues that simply crash rizin or one of the other tools (e.g. NULL pointer dereferences). Issues that can leak non-sensitive memory contents (e.g. memory addresses/memory values that are not files, private keys, etc.) will be considered on a case-by-case basis and they should be demonstrated to be somehow usable by an attacker.
```
Comment (XVilka) [inline SECURITY.md]: I would add it's a `user-space tool` and **not a service**.

**Review by XVilka — CHANGES_REQUESTED** [review]

**Review by XVilka — APPROVED** [review]

File: SECURITY.md [hunk 2]
```diff
@@ -19,3 +19,5 @@ If you have not received a reply to your email within 48 hours, or have not hear
 - Inform the team over the [public chats](https://rizin.re/#community) that you sent a message regarding a security issue.
 
 **Important:** Don't disclose any information regarding the issue itself in the public chats.
+
+While analysing reports, we consider the main usage of Rizin nowadays as a user-space Command Line Interface (CLI) tool and not as a service or as a library. For this reason the security team does not consider as vulnerabilities those issues that simply crash rizin or one of the other tools (e.g. NULL pointer dereferences, memory exhaustion or in general those flaws that only impact a service availability). Issues that can leak non-sensitive memory contents (e.g. memory addresses/memory values that are not files, private keys, etc.) will be considered on a case by case and they should be demonstrated to be somehow usable by an attacker. Please try to always include in your report an attack scenario, showing how the issue can affect Rizin users.
```
Comment (XVilka) [inline SECURITY.md]: `considered on a case by case` -> `will be considered case by case`

Comment (wargio) [inline SECURITY.md:23]:
```suggestion
While analysing reports, we consider the main usage of Rizin nowadays as a user-space Command Line Interface (CLI) tool and not as a service or as a library. For this reason the security team does not consider as vulnerabilities those issues that simply crash rizin or one of the other tools (e.g. NULL pointer dereferences, memory exhaustion or in general those flaws that only impact a service availability). Issues that can leak non-sensitive memory contents (e.g. memory addresses/memory values that are not files, private keys, etc.) will be considered case by case and they should be demonstrated to be somehow usable by an attacker. Please try to always include in your report an attack scenario, showing how the issue can affect Rizin users. 
```

**Review by ITAYC0HEN — APPROVED** [review]
approved after my commit. Good work

File: SECURITY.md [hunk 3]
```diff
@@ -1,17 +1,23 @@
...
 
-## Supported Versions
+## Scope and Supported Versions
 
 | Version          | Supported          |
 | ---------------- | ------------------ |
 | latest-release   | :white_check_mark: |
 | *                | :x:                |
 
+Rizin is a user-space Command Line Interface (CLI) tool which isn't considered a high-availability service. As such, issues that impact Rizin's availability will not be considered as vulnerbilities by the security team. Such issues include, but not limitied to:
```
Comment (XVilka) [inline SECURITY.md]: vulnerabilities

File: SECURITY.md [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
+ - NULL Pointer Dereference
+ - Memory Exhaustion
+
+Issues that can leak non-sensitive memory contents (e.g memory addresses and values, etc.) shuld be demonstrated to be usable by an attacker, and will be considered on a case-by-case basis.
```
Comment (XVilka) [inline SECURITY.md]: should

Comment (XVilka) [reply SECURITY.md]: Please no `case-by-case basis`, it's a poor style.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 3 above)
Comment (ITAYC0HEN) [reply SECURITY.md]:
```suggestion
Rizin is a user-space Command Line Interface (CLI) tool that isn't considered a high-availability service. As such, issues that impact Rizin's availability will not be considered as vulnerabilities by the security team. Such issues include, but are not limited to:
```

File: SECURITY.md (hunk 4 above)
Comment (ITAYC0HEN) [reply SECURITY.md]:
```suggestion
Issues that can leak non-sensitive memory contents (e.g memory addresses and values, etc.) should be demonstrated to be usable by an attacker and will be considered on a case-by-case basis.
```

**Review by XVilka — APPROVED** [review]

**Review by wargio — APPROVED** [review]

**Pull request merged**